components/brave_wallet/brave_wallet_utils.cc

```diff
@@ -133,4 +137,48 @@ std::string Uint256ValueToHex(uint256_t input) {
   return out;
 }
 
+std::string GenerateMnemonic(const std::vector<uint8_t>& entropy) {
+  uint8_t* entropy_in = const_cast<uint8_t*>(entropy.data());
+  size_t size_in = entropy.size();
+
+  // If input entropy is empty, we generate a random 128 bits
```

Comment:
Why would we allow for external entropy? Id rather ensure that code calling this is forced to generate its own entropy via an audited method here. Allowing external entropy is a recipe for insecure random number generation.

Reply:
I suppose one reason could be to allow the caller of this method to specify how long the mnemonic should be, but I would still prefer we expose a "number of bits" parameter (or bytes), each value being a 32bit increment as per the BIP39 spec (that would allow for the following 5 options: 128, 160, 192, 224, 256) rather than letting the caller generate the entropy themselves.

Reply:
It was a convenient interface for specifying entropy in tests. It's now fixed in ca4d806d885cb83113b7163a9861ffdbfde28203